### PR TITLE
PSG-3981

### PR DESCRIPTION
**Pull request by SinaSeylani**
- added ChangeContactTests
- added CurrentUserTests
- created a new KEY ("OTP_TEST_USER_AUTH_TOKEN") and added it to the Github actions secrets and 1password passage vault.

### CODE REVIEW

File: passage/src/androidTest/java/id/passage/android/ChangeContactTests.kt [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            // make sure we have an authToken.
+            assertNotEquals(AUTH_TOEKN, "")
+            try {
+                Passage.setAuthToken(AUTH_TOEKN)
+                val date = System.currentTimeMillis()
+                val identifier = "authentigator+$date@passage.id"
+                val response = CURRENT_USER.changeEmail(identifier)
+                assertNotNull(response.id)
+            } catch (e: Exception) {
+                assertTrue(false)
```
Comment (rickycpadilla) [inline passage/src/androidTest/java/id/passage/android/ChangeContactTests.kt:50]: For failing a test, lets use the `fail` method, as its a bit easier to read.
```suggestion
                fail("Test failed due to unexpected exception: ${e.message}")
```

File: passage/src/androidTest/java/id/passage/android/IntegrationTestConfig.kt [hunk 2]
```diff
@@ -9,5 +12,20 @@ internal class IntegrationTestConfig {
...
+                    emailVerified = true,
+                    id = "Cf70Zkwn1ywnpuua1ZFF717G",
+                    phone = "",
+                    phoneVerified = false,
+                    status = UserStatus.active,
+                    userMetadata = null,
+                    webauthn = false,
+                    webauthnTypes = emptyList(),
+                ),
+            )
```
Comment (rickycpadilla) [inline passage/src/androidTest/java/id/passage/android/IntegrationTestConfig.kt:29]: I believe you should be able to init the PassageUser instance directly? Maybe like:
```suggestion
            PassageUser(
                  email = "authentigator+1716916054778@ncor7c1m.mailosaur.net",
                  emailVerified = true,
                  id = "Cf70Zkwn1ywnpuua1ZFF717G",
                  phone = "",
                  phoneVerified = false,
                  status = UserStatus.active,
                  userMetadata = null,
                  webauthn = false,
                  webauthnTypes = emptyList(),
            )
```

**Review by rickycpadilla — COMMENTED** [review]
Thanks for adding these tests! 🎉 A couple very minor change requests.

File: passage/src/androidTest/java/id/passage/android/ChangeContactTests.kt (hunk 1 above)
Comment (rickycpadilla) [reply passage/src/androidTest/java/id/passage/android/ChangeContactTests.kt:50]: This applies in other tests in the file as well.

File: passage/src/androidTest/java/id/passage/android/IntegrationTestConfig.kt (hunk 2 above)
Comment (SinaSeylani) [reply passage/src/androidTest/java/id/passage/android/IntegrationTestConfig.kt:29]: Since 'PassageUser' has private constructor, it's not possible

**Review by rickycpadilla — APPROVED** [review]
Looks great. Ship it! 🚀

**Pull request merged**